src/app/pages/poll/poll.component.ts

```diff
@@ -91,12 +99,15 @@ export class PollComponent {
    * @param url Customized input url
    */
   onSlugChange(url: string): void {
-    if (!url || url.length === 0) return;
+    if (!url || url.length === 0) {
+      return;
+    }
     const oldSlug = this.poll.slug;
     this.poll.slug = encodeURI(url.toLocaleLowerCase().split(' ').join('-'));
 
     // Commit changes to the server
-    this.pollService.update(this.poll).subscribe((_) => {}, (_) => {
+    this.pollService.update(this.poll).subscribe((_) => {
+    }, (_) => {
```

Comment:
how does that work? don't get it

Reply:
the nested `(_) => {...}`

Reply:
We ignore the output here. I tried to replace that with `null`, but that override of the `subscribe` method seems to be deprecated.